lms/templates/fields/field_dropdown.underscore

```diff
@@ -40,6 +40,13 @@
     <% } %>
 </span>
 
+<% if (editable != 'never' && id === 'account_privacy') { %>
+    <button class="btn btn-change-privacy" aria-live="polite">
+        <span><%- gettext('Save') %></span>
+        <span class="sr"><%- gettext('Click to save') %></span>
```

Comment:
@waheedahmed the additional context is not necessary here.  "Save" is enough for all users to understand the purpose of the button.  I'm not sure the aria-live is necessary either unless the value of the button gets changed dynamically.  It is also not likely to work since the button gets added dynamically.

Reply:
@cptvitamin Done, but screen reader will not read a button is added to the DOM without this, is that okay?

Reply:
@waheedahmed its actually not reading it now either since the div with the aria-live on it cannot itself be added dynamically.  It needs to be present when the page loads so that they screen reader knows to watch it for changes.  That said, its not necessary to announce that the save button has been added since it appears logically in the tab order following the select menu.  All that is required that it announces it self when it receives focus, and it does that.  Thanks for checking!